contrib/opentimelineio_contrib/application_plugins/rv/example_otio_reader/otio_reader.py

```diff
@@ -206,6 +206,18 @@ def _create_media_reference(item, track_kind=None):
                 # Appending blank to media promotes name of audio file in RV
                 media.append(blank)
 
+            return media
+        elif isinstance(item.media_reference,
+                        otio.schema.ImageSequenceReference):
+            frame_sub = "%0{n}d".format(
+                n=item.media_reference.frame_zero_padding
+            )
+
+            media = [
+                str(item.media_reference.abstract_target_url(symbol=frame_sub))
+            ]
+            print("ImageSequenceReference media: %s" % media)
```

Comment:
Is it desirable to print from within an adapter?

Reply:
Ack! Debug. I'll remove. Good catch.